### PR TITLE
Feature/introduce seances

### PR DESCRIPTION
**Pull request by MonikaLukowska**
In this PR I introduce seances.  It covers:

- adding model, controller and views
- adding associations to movie and hall
- adding system and model tests

User can:

- create, edit and destroy seance
- lists seances grouped by movies from current date
- navigate to seances from the 6 following days

https://user-images.githubusercontent.com/56922072/191046589-052c5be9-cbf9-4f96-b8b2-5494638b3e30.mp4

### CODE REVIEW

File: app/controllers/seances_controller.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    else
+      render :new, locals: { seance: record }
+    end
+  end
+
+  def update
+    if seance.update(seance_params)
+      redirect_to seance, notice: t('.notice')
+    else
+      render :edit
```
Comment (baniak66) [inline app/controllers/seances_controller.rb]: I think you miss locals here. Please check also `movies#update` action.

File: app/validators/availability_validator.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class AvailabilityValidator < ActiveModel::EachValidator
```
Comment (baniak66) [inline app/validators/availability_validator.rb:1]: I like your custom validator, but there is an issue. When I try to edit the price of the existing seance I get `Start time not available` error.

**Review by baniak66 — COMMENTED** [review]
Nice job 👏 Just small remarks.

File: app/controllers/seances_controller.rb (hunk 1 above)
Comment (MonikaLukowska) [reply app/controllers/seances_controller.rb]: you're right, I should pass locals here as well

File: app/validators/availability_validator.rb (hunk 2 above)
Comment (MonikaLukowska) [reply app/validators/availability_validator.rb:1]: Didn't think about it. Record should be excluded. Thanks:)

**Review by baniak66 — APPROVED** [review]

**Pull request merged**